src/crimson/common/interruptible_future.h

```diff
@@ -830,6 +862,14 @@ struct interruptor
 	      seastar::future<>>(seastar::now());
   }
 
+  template <typename ValueT = void, typename A>
+  [[gnu::always_inline]]
+  static interruptible_future_detail<InterruptCond, seastar::future<ValueT>>
+  make_ready_future(A&& value) {
+    return interruptible_future_detail<InterruptCond, seastar::future<ValueT>>(
+	seastar::make_ready_future<ValueT>(std::move(value)));
+  }
```

Comment:
```suggestion
  template <typename ValueT = void, typename... A>
  [[gnu::always_inline]]
  static interruptible_future_detail<InterruptCond, seastar::future<ValueT>>
  make_ready_future(A&&... value) {
    return interruptible_future_detail<InterruptCond, seastar::future<ValueT>>(
    seastar::make_ready_future<ValueT>(std::forward<A>(value)...));
  }
```

for the same reason why i propose https://github.com/ceph/ceph/pull/41303/commits/6a1f3136178a3a02f8d2743643fb46950224140c

Reply:
Um, this is exactly what i did previously.

Reply:
no, you packed the parameter as a `tuple<>` instead of perfect forwarding them.

Reply:
done:-)